src/mui/field/PhoneField.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash.get';
+import pure from 'recompose/pure';
+
+import { parse, format } from 'libphonenumber-js';
+
+export const localPhoneFormat = (input, locale) => {
```

Comment:
avoid meaningless variable name as input/output/string/...
parsed_number would be nice in this case.